context_request.go

```diff
@@ -433,15 +433,15 @@ func getPostData(ctx *Context) (map[string]any, error) {
 	contentType := ctx.instance.Get("Content-Type")
 	data := make(map[string]any)
 
-	if contentType == "application/json" {
+	if strings.Contains(contentType, "application/json") {
 		bodyBytes := ctx.instance.Body()
 
 		if err := json.Unmarshal(bodyBytes, &data); err != nil {
 			return nil, fmt.Errorf("decode json [%v] error: %v", string(bodyBytes), err)
 		}
 	}
 
-	if contentType == "multipart/form-data" {
+	if strings.Contains(contentType, "multipart/form-data") {
```

Comment:
The `ContentType` is `multipart/form-data; b*****`, so we can't simply judge equality. Gin doesn't have the issue.